### ISSUE TITLE
[Bug]:  Create Tour within a Resource 

### ISSUE THREAD

**Issue by ingmontoya** [JibayMcs/filament-tour#9]
### What happened?

is there a way to trigger/open a tour within a Filament resource ? 

### How to reproduce the bug

Expected behaviour: Open a tour within a filament resources

### Package Version

3.0

### PHP Version

8.2

### Laravel Version

10.0

### Which operating systems does with happen with?

macOS

**Comment by JibayMcs** [comment]
Hi !

I don't quite understand the request, 

Instead of having to define `HasTour` in each resource pages (List/Create/Edit,etc)

You'd like to be able to manage it directly in the resource, right?

**Comment by Mrkbingham** [comment]
@ingmontoya I don't think this is a bug.  You should be able to use the `HasTour` trait on any resource page.  You may have been missing the piece that I've added to this PR: https://github.com/JibayMcs/filament-tour/pull/27

The short version is that the docs currently provide examples of user-initiated tours.  If you check out the new sample section I've added, you can see instructions on how to get the Tour to load immediately after the page is loaded.  Hope this helps!

**Comment by Mrkbingham** [comment]
Quick update - I was wrong earlier, it turns out the missing piece on my issue was that I wasn't using the correct route setting on my tours, as that's the default supported functionality for rendering tours that can also account for the only_visible_once setting
